code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -564,12 +564,13 @@
 		if(CHARGE_CRUSH)
 			Paralyze(CHARGE_SPEED(charge_datum) * 2 SECONDS)
 		if(CHARGE_BULL_HEADBUTT)
-			Paralyze(CHARGE_SPEED(charge_datum) * 2 SECONDS)
+			Paralyze(CHARGE_SPEED(charge_datum) * 1.5 SECONDS)
 		if(CHARGE_BULL)
 			Paralyze(CHARGE_SPEED(charge_datum) * 0.2 SECONDS)
 		if(CHARGE_BULL_GORE)
 			adjust_stagger(CHARGE_SPEED(charge_datum) * 1 SECONDS)
-			reagents.add_reagent(/datum/reagent/toxin/xeno_ozelomelyn, 10)
+			adjust_slowdown(CHARGE_SPEED(charge_datum) * 1)
+			reagents.add_reagent(/datum/reagent/toxin/xeno_neurotoxin, 30)
```

Comment:
On one hand 30u neuro is insane, but 10u ozy is arguably more powerful and balanced. Getting a gore charge off on someone twice is not hard and 60u neuro is extremely debilitating.

Reply:
It's almost 2 prae spits worth (lesser since prae also leaves a gas cloud). And I think staying in the Bulls charging lane after getting gored once can be chalked up to an issue of skill.

Only place I can see it being too strong is when xenos surround a marine inside maze, making the gore charges guaranteed. But then again, xenos surrounding marine is usually death within seconds anyways.

Then again, I'm not an expert on neuro. I didn't even know Prae did 15u before making this pr. Maybe 25u would be better?